### DISCUSSION TITLE
Bug Report: Сustom circuit boxes for some reason get locked by lock default wiring options

### DISCUSSION THREAD

**Discussion by SerChiz** [FakeFishGames/Barotrauma#13979] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Сustom circuit boxes for some reason get locked by lock default wiring options and neither original nor custom circuit boxes actually unlock after removing loсk default wiring checkmark.

![screen](https://github.com/FakeFishGames/Barotrauma/assets/82869566/7531a033-a504-4d46-bfe9-071df1c27337)

### Reproduction steps

1 Multiplayer campaign.
2. Set settings options "lock default wiring" to true
2 Сraft new Circuit Box
3 Place the Circuit Box on wall
4 Created a schematic in it
5 Play level and restart server
6 Custom Circuit Box is lock!

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by microwave805** [comment]
Same issue. this is really annoying, as i have to do my circuits within one round

**Comment by Jasontti** [comment]
Tested the fix for this, no issues found.
Closing discussion as completed.

**Reply by dossy** [reply]
What was the fix for this?

I just experienced this in a Linux DedicatedServer Vanilla campaign: a player-created circuit box somehow ended up locked, and there's no obvious way to unlock it within the game itself.

Do I have to jump through the hoops of extracting the .sub file from the multiplayer .save file, editing it in the sub editor to unlock the box, then put the modified .sub back in the .save file, in order to "fix" this?

**Reply by microwave805** [reply]
> What was the fix for this?
> 
> I just experienced this in a Linux DedicatedServer Vanilla campaign: a player-created circuit box somehow ended up locked, and there's no obvious way to unlock it within the game itself.
> 
> Do I have to jump through the hoops of extracting the .sub file from the multiplayer .save file, editing it in the sub editor to unlock the box, then put the modified .sub back in the .save file, in order to "fix" this?

Yup you do. There may also be a mod that lets you unlock wiring.
